modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml

```diff
@@ -299,6 +299,41 @@
 		<clean-database-upgrade-client />
 	</target>
 
+	<target name="check-upgrade-client-home-properties">
+		<get-database-upgrade-client-paths />
+
+		<prepare-database-upgrade-configuration />
+
+		<property name="user.home" value="~" />
+
+		<execute dir="${user.home}" failonerror="true">
+			touch portal-ext.properties
+		</execute>
+
+		<execute dir="${db.upgrade.client.home}" failonerror="true">
+			db_upgrade${file.suffix.bat} run > temp.log
+		</execute>
+
+		<loadfile
+			property="upgrade.process.output"
+			srcFile="temp.log"
+		/>
+
+		<echo>${upgrade.process.output}</echo>
+
+		<if>
+			<contains string="${upgrade.process.output}" substring="Remove /home/${user.name}/portal-ext.properties prior to running an upgrade to prevent possible conflicts." />
```

Comment:
Change substring to "Remove ${user.home}/portal-ext.properties prior to running an upgrade to prevent possible conflicts."

Reply:
Looks like [CI's output](https://testray.liferay.com/reports/production/logs/2022-11/test-1-30/test-portal-acceptance-pullrequest(master)/5470/functional-upgrade-tomcat90-mysql57-jdk8/0/7/jenkins-console.txt.gz?authuser=0) is "Remove /root/portal-ext.properties prior to running an upgrade to prevent possible conflicts."

On CI, it seems to just be _/root_ instead of _/home/root_, so I think `${user.home}` is better than `${user.name}`. 

`${user.home}` covers _/home/user/_ for me locally.